lib/datasets/wine.rb

```diff
@@ -0,0 +1,64 @@
+require 'csv'
+
+require_relative 'dataset'
+
+module Datasets
+  class Wine < Dataset
+    Record = Struct.new(:class,
+                        :alcohol,
+                        :malic_acid,
+                        :ash,
+                        :alcalinity_of_ash,
+                        :n_magnesium,
```

Comment:
The last `s` is missing: `n_magnesiums`. (`n_magnesium_atoms` may be better.)

Reply:
Sure.

Reply:
Fixed